### PR TITLE
Muzima-688: Design and Develop mUzima Help Feature

### PR DESCRIPTION
**Pull request by rezita**

### CODE REVIEW

**Comment by samuelowino** [comment]
Hi Zita change this pull request to point to muzima:outreachy-zita branch

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -640,4 +640,24 @@ Ces formulaires avant de télécharger une version plus récente. </string>
     <string name="warning_switch_server">Changement de serveur: Cela effacera toutes les données mUzima sur votre appareil, y compris les formulaires incomplets. Etes-vous sûr de vouloir modifier ce paramètre?</string>
     <string name="write_SHR_to_card_text">Ecrire SHR à la carte</string>
     <string name="write_shr_to_card_general">Ecrire des dossiers de santé partagés à la carte</string>
+    <string name="feedback_header">S\'il vous plaît envoyez-nous vos commentaires et réactions</string>
+    <string name="feedback_rating_question1">Comment facile à utiliser mUzima App?</string>
```
Comment (benamonya) [inline app/src/main/res/values-fr/strings.xml]: Hi Zita, rename feedback_rating_question1 to feedback_rating_question

Comment (rezita) [reply app/src/main/res/values-fr/strings.xml]: Hello Benard, I have changed it.

Comment (benamonya) [reply app/src/main/res/values-fr/strings.xml]: Hi @rezita, Rename in the other strings.xml

Comment (rezita) [reply app/src/main/res/values-fr/strings.xml]: Hello @benamonya, I did it.

**Pull request merged**